package.json

```diff
@@ -23,7 +23,8 @@
     "rdf-isomorphic": "^1.3.0",
     "rdf-string": "^1.6.0",
     "rdf-terms": "^1.10.0",
-    "sparqljs": "^3.7.1"
+    "sparqlalgebrajs": "^4.3.7",
```

Comment:
Don't make it depend on itself.

Reply:
Changes to the variables made as you've suggested. Also removed the self-dependency